src/clj_jargon/users.clj

```diff
@@ -59,12 +59,12 @@
         :else false))
 
 (defn list-groups
-  "List groups (qualified usernames), using an optional search (postgresql/iquest LIKE format)"
+  "List groups (qualified usernames), using an optional search string"
   ([{^UserGroupAO ug-ao :userGroupAO}]
    (for [^UserGroup ug (.findAll ug-ao)]
      (str (.getUserGroupName ug) "#" (.getZone ug))))
-  ([{^UserGroupAO ug-ao :userGroupAO} search]
-   (for [^UserGroup ug (.findWhere ug-ao search)]
+  ([{^UserGroupAO ug-ao :userGroupAO} search & {:keys [case-insensitive?] :or {case-insensitive? false}}]
+   (for [^UserGroup ug (.findUserGroups ug-ao search case-insensitive?)]
```

Comment:
`findWhere` no longer exists. I couldn't find any uses of `list-groups` in our code, so I updated it to use the slightly different `findUserGroups`, which takes a simple search string instead of a WHERE clause.